api/cloud/oc_cloud_apis.c

```diff
@@ -698,6 +699,26 @@ cloud_access_refresh_access_token(oc_endpoint_t *endpoint, const char *uid,
 
   return oc_do_post();
 }
+
+bool
+cloud_send_ping(oc_endpoint_t *endpoint, uint16_t timeout_seconds,
+                oc_response_handler_t handler, void *user_data)
+{
+  if (!oc_tls_connected(endpoint)) {
```

Comment:
Every other call to `oc_tls_connected` seems to be under OC_SECURITY macro. Isn't it missing here?

Reply:
fixed